terminal.sh

```
@@ -1 +1 @@
-RUST_LOG=cosmic_text=debug,terminal=debug cargo run --release --package terminal -- "$@"
+RUST_LOG=fastui_cosmic=debug,terminal=debug cargo run --release --package terminal -- "$@"
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)